### DISCUSSION TITLE
Change list of the Combo Box

### DISCUSSION THREAD

**Discussion by DyNgPinasOfficial** [TomSchimansky/CustomTkinter#379] [Q&A]
How could I update or change the dropdown list of the combo box? I have tried different ways based on tkinter, but it doesnt work for me.

**Comment by abhaymakes** [comment]
Use values argument in the CTkComboBox (`values=["option 1", "option 2"]`)

**Reply by DyNgPinasOfficial** [reply]
What I meant is that I want to update those values from other method and I had tried to update the CTkComboBox by changing the values from other method but it does not work. I'm sorry for not clarifying it. However, I found a way to update it. My new question is; Is it efficient? 

What I have done is just call the method to redisplay the UI it contains.
![image](https://user-images.githubusercontent.com/50480977/184046283-70c74420-7aa3-48d8-a18d-560db05df110.png)
